diff --git a/dist/generator.js b/dist/generator.js
index c2ba1224e6ec7c3cd71bcfff7b2a457516a5174c..b1ad10f51ec2cf08ca42ef6ebbb40829b80d2342 100644
--- a/dist/generator.js
+++ b/dist/generator.js
@@ -66,7 +66,7 @@ export function generateSpec(options) {
                     openapi: '3.0.0',
                     info: baseSpec.info
                 },
-                apis: [apiPath],
+                apis: [apiPath.replaceAll('[', '\\[').replaceAll(']', '\\]')],
                 failOnErrors
             });
             // Check if the file actually has any swagger documentation
